.changeset/young-carpets-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
```

Comment:
`minor` for this one imo, since we're introducing a new feature and it's 1.0

Reply:
Resolved in [ca9c51b](https://github.com/backstage/backstage/pull/28971/commits/ca9c51ba1bb30ea326e323c37e1ba469222a2452)